flask/app/models.py

```diff
@@ -341,6 +341,11 @@ class AnalysisState(str, Enum):
     Cancelled = "Cancelled"
 
 
+class PriorityType(str, Enum):
+    ClinicalPriority = "ClinicalPriority"
+    ResearchPriority = "ResearchPriority"
```

Comment:
Including `Priority` in the value of the enum seems kind of redundant when both the column and the enum type are labelled as such.

Reply:
This also changes the number of enumerations available in `/api/enums`

Reply:
fixed in e635324d3c3643d525731b5628c95a5dcd529723